File: SecondYear/any-mesh_AnimeshKumarSingh_2226CSE(AI&ML)1119_2/Week_2/Fibbonacci.cpp
vector<int> generateFibonacciNumbers(int n) {
    // Write your code here.
    vector<int> ans;
    int a = 0;
    ans.push_back(a);
    if(n == 1)
        return ans;
    int b = 1;
    ans.push_back(b);
    if(n == 2)
        return ans;
    for(int i = 3 ; i <= n ; i++){
        int c = a + b;
        ans.push_back(c);
        a = b;
        b = c;
    }
    return ans;
}

File: SecondYear/aayushker_AayushkerSingh_2226CSE(AI)1010_2/Week_2/Fibbonacci.py
from typing import List

def generateFibonacciNumbers(n: int) -> List[int]: 
    # Write your code here
    a,b,i=-1,1,1
    l=[]
    while(i<=n):
            c=a+b       
            a=b
            b=c
            l.append(c)
            i+=1

    return l
    pass